sparse-11/usr/libexec/droid-hybris/system/etc/init/disabled_services.rc

```diff
@@ -0,0 +1,29 @@
+service netd netd_HYBRIS_DISABLED
+
+service installd installd_HYBRIS_DISABLED
+
+service cameraserver cameraserver_HYBRIS_DISABLED
+
+service audioserver audioserver_HYBRIS_DISABLED
+
+service media mediaserver_HYBRIS_DISABLED
+
+service update_engine update_engine_HYBRIS_DISABLED
+
+service vendor.audio-hal-2-0 /vendor/bin/hw/android.hardware.audio@2.0-service_HYBRIS_DISABLED
```

Comment:
Is it really audio hal 2.0?

Reply:
The service was renamed, wondering if this even should be here, this seems vendor specific, although probably in most cases the same.